c2_components/src/mfx_c2_encoder_component.cpp

```diff
@@ -278,6 +286,36 @@ void MfxC2EncoderComponent::RegisterClass(MfxC2ComponentsRegistry& registry)
         &MfxC2Component::Factory<MfxC2EncoderComponent, EncoderType>::Create<ENCODER_VP9>);
 }
 
+void MfxC2EncoderComponent::getMaxMinResolutionSupported(
```

Comment:
We can get the supported size from media_codecs_intel_c2_video.xml by MfxXmlParser. Why must have to add this function?

Reply:
This is a good concern. Yes, We can get the max and min video size from the config file. But this is for MediaCodec API.

For this vts test, it requested max and min video size via codec2.0 API. If the app cannot query it, the vts will fail. Please check the details in below link.
http://inlubt0232.iind.intel.com:8080/source/xref/CELADON/frameworks/av/media/codec2/hidl/1.0/vts/functional/video/VtsHalMediaC2V1_0TargetVideoEncTest.cpp?r=15ccfa68#461